modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -325,30 +330,34 @@ export default function ObjectFieldFormBase({
 				/>
 			)}
 
-			{picklistBusinessType && (
-				<Select
-					disabled={disabled}
-					error={errors.listTypeDefinitionId}
-					label={Liferay.Language.get('picklist')}
-					onChange={({target: {value}}) => {
-						setValues({
-							defaultValue: '',
-							listTypeDefinitionExternalReferenceCode: pickLists.find(
-								({name}) => name === value
-							)?.externalReferenceCode,
-							listTypeDefinitionId: Number(
-								pickLists.find(({name}) => name === value)?.id
-							),
-							state: false,
-						});
-					}}
-					options={pickLists}
-					required
-					value={
-						validListTypeDefinitionId ? selectedPicklist?.name : ''
-					}
-				/>
-			)}
+			{values.businessType === 'Picklist' ||
+				(values.businessType === 'MultiselectPicklist' && (
```

Comment:
Hi @danvb112. I think this way can make it more clear:

```suggestion
			{(values.businessType === 'Picklist' ||
				values.businessType === 'MultiselectPicklist') && (
```

Reply:
Thanks! @alinedoleron